app/models/status.rb

```diff
@@ -256,7 +256,7 @@ def non_sensitive_with_media?
   end
 
   def reported?
-    @reported ||= Report.where(target_account: account).unresolved.where('? = ANY(status_ids)', id).exists?
+    @reported ||= Report.where(target_account: account).where('? = ANY(status_ids)', id).exists?
```

Comment:
We want to change the definition of `status.reported?` to be true even if the Report against the Status has already been resolved.

This is because the Admin Status Policy only gives elevated read access to a private Status if `reported?` is true. And we want admins to still have access to private Statuses that were reported, even after the report is resolved (for auditing purposes).